publisher/publisher.go

```diff
@@ -130,17 +123,46 @@ type ctSubmissionRequest struct {
 	Chain []string `json:"chain"`
 }
 
+type pubMetrics struct {
+	submissionLatency *prometheus.HistogramVec
+	errors            *prometheus.CounterVec
+}
+
+func initMetrics(stats metrics.Scope) *pubMetrics {
+	submissionLatency := prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name: "ct_submission_latency",
```

Comment:
I'd call this "submission_time_seconds" to match our other metrics. And I'd suggest including "status" as an additional label, and removing the separate errors count. That way we get a total submission count, plus an error count, for free.